owncloud-android-library

```diff
@@ -1 +1 @@
-Subproject commit 942971a202f7404b135b26795c1b545c390a023f
+Subproject commit 8c4834256b7d4f44822570973c03164d84586264
```

Comment:
Please, upgrade to commit in android-library/develop after merging license update.